core/domain/subscription_services_test.py

```diff
@@ -56,6 +57,20 @@ def setUp(self):
         self.viewer_id = self.get_user_id_from_email(self.VIEWER_EMAIL)
         self.owner_2_id = self.get_user_id_from_email(self.OWNER_2_EMAIL)
 
+    def _get_all_subscribers_of_creators(self, user_id):
```

Comment:
I'm convinced by your argument; let's leave it as-is. Thanks for the analysis, and sorry for the churn!